example/test/function_component_test.dart

```diff
@@ -0,0 +1,34 @@
+import 'dart:html';
+
+import 'package:react/react.dart' as react;
+import 'package:react/react_dom.dart' as react_dom;
+import 'package:react/react_client.dart';
+
+import 'react_test_components.dart';
+
+void main() {
+  setClientConfiguration();
+  var inputValue = 'World';
+  // TODO: replace this with hooks/useState when they are added.
```

Comment:
Can we add a note / AC item to make sure this gets addressed via #223?

Reply:
@sydneyjodon-wk already addressed this in her pr